omod/src/main/webapp/resources/htmlforms/mexico/consult.xml

```diff
@@ -193,156 +197,115 @@
                         </td>
                         <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:SYSTOLIC BLOOD PRESSURE').obsDatetime"/>
-                            </p>
+                            <lookup expression="fn.latestObs('PIH:SYSTOLIC BLOOD PRESSURE').obsDatetime"/>
                         </td>
                         <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:SYSTOLIC BLOOD PRESSURE').valueNumeric"/>
-                            </p>
-                        </td>
-                    </tr>
-                    <tr>
-                        <td>
-                            <uimessage code="pihcore.vitals.diastolic.bp.title"/>
-                        </td>
-                        <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:DIASTOLIC BLOOD PRESSURE').obsDatetime"/>
-                            </p>
-                        </td>
-                        <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:DIASTOLIC BLOOD PRESSURE').valueNumeric"/>
-                            </p>
+                            <lookup expression="fn.latestObs('PIH:SYSTOLIC BLOOD PRESSURE').valueNumeric.intValue()"/>
+                            <lookup complexExpression="#if($fn.latestObs('PIH:SYSTOLIC BLOOD PRESSURE')) / #end" />
+                            <lookup expression="fn.latestObs('PIH:DIASTOLIC BLOOD PRESSURE').valueNumeric.intValue()"/>
                         </td>
                     </tr>
                     <tr>
                         <td>
                             <uimessage code="mirebalais.vitals.o2sat.title"/>
                         </td>
                         <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:BLOOD OXYGEN SATURATION').obsDatetime"/>
-                            </p>
-                        </td>
-                        <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:BLOOD OXYGEN SATURATION').valueNumeric"/>
-                            </p>
-                        </td>
-                    </tr>
-                    <tr>
-                        <td>
-                            <uimessage code="pihcore.lab.isFastingGlucose"/>
-                        </td>
-                        <td>
-                            <p>
-                                <lookup expression="fn.latestObs('PIH:Fasting for blood glucose test').obsDatetime"/>
-                            </p>
+                            <lookup expression="fn.latestObs('PIH:BLOOD OXYGEN SATURATION').obsDatetime"/>
                         </td>
                         <td>
-                            <p>
```

Comment:
`<p>` tags not needed within `<td>` -- here they just create some additional vertical spacing, which I don't think is needed.